plugin.py

```diff
@@ -31,7 +31,9 @@ def plugin_info(self):
         return plugin_info
 
     def process(self, trace, data_context):
-        kaitai_utils.write_kaitai_to_trace(trace)
+        bytearray_length = 64
```

Comment:
Maybe make it clear that this value can be adjusted with a comment?

Reply:
Good idea, done.